docs/struct.md

```diff
@@ -39,6 +39,7 @@ It extends the [Extend](#extend) and [Stage](#stage) structures.
 | `builders` | map<string, [Stage](#stage)> | The builder stages of the Dockerfile. |
 | `entrypoint` | string[] | The entrypoint of the Dockerfile. See [Dockerfile reference](https://docs.docker.com/reference/dockerfile/#entrypoint). |
 | `cmd` | string[] | The default command of the Dockerfile. See [Dockerfile reference](https://docs.docker.com/reference/dockerfile/#cmd). |
+| `volume` | string[] | Create volume mounts. See [Dockerfile reference](https://docs.docker.com/reference/dockerfile/#volume). |
```

Comment:
Maybe specify that the `volume` instruction does not support specifying a host-dir parameter. It's only `/srv/app` for example.

Reply:
I don't think that it's the purpose of our documentation. If they want more details, they can refer to the Dockerfile documentation (at least for now).